packages/pwa-kit-cli/src/ssr/server/build-dev-server.js

```diff
@@ -100,6 +99,7 @@ export const DevServerMixin = {
                 console.log(chalk.cyan('First build complete'))
             }, 75)
         })
+        app.__hotServerMiddleware = webpackHotServerMiddleware(app.__compiler)
```

Comment:
Moving this line from the old `addSSRRenderer` to here in `addSDKInternalHandlers` makes it possible to break the local dev server. Seeing this error when `npm start` from within the template-express-minimal:

```
/Users/vmarta/repos/pwa-kit-copy/packages/pwa-kit-cli/node_modules/webpack-hot-server-middleware/src/index.js:117
        throw new Error(`Expected a webpack compiler named 'server'`);
```

Why is the error happening? Because now webpack hot server middleware is attached for every template, while it used to be attached _only_ when `addSSRRenderer()` is called.

Reply:
Incredibly well spotted – thank you so much!